### PR TITLE
fix homepage style

### LINKED ISSUE #18: Home page style
Fixing the style of the home page

- [ ] Navbar Style & Max-width (for the entire screen because we have a big  white space between the nav and the main content & the categories are not in the same size whit Featured Courses )
- [ ] Change bookmarks to Ceren Bookmarks

### PR DESCRIPTION
**Pull request by vuslatsena**
I'm opening a pr so that it doesn't get lost. I'll mark it as I progress.

- [x] Navbar Style:  Big white space between the nav and the main content
- [x] The categories size fix
- [x]  Bookmarks to Ceren Bookmarks
- I added a show more/less button instead of compressing the card sizes. I've run out of solutions :')

Extra changes

- Created assets folder for svgPath because  It looked messy in the NavBar file. We can keep the svgpath or images in this file)
- Netlify deploy error when the file name was change.(navBar to NavBar) I change the name of the NavBar and put it in the folder. Now no errors💆🏼‍♀️ 
- The NavBar had a lot of code duplication, I simplified it by using the map function for the navbar elements
fixes #18

### CODE REVIEW

File: components/home-page-components/RegisteredCourseCard.css [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/* I think this styles just for bookmark maybe we can change the name or 
+create a bookmark component */
+
```
Comment (wasimtayyan) [inline components/home-page-components/RegisteredCourseCard.css:3]: we can change the name

**Review by wasimtayyan — COMMENTED** [review]
I really want to give you feedback but I didn't see any change in categories section since we had last conversation in Discord

**Comment by vuslatsena** [comment]
> I really want to give you feedback but I didn't see any change in categories section since we had last conversation in Discord

yes yes i am trying the suggested styles, i will post an update here before the sprint is over

**Comment by vuslatsena** [comment]
it seems okay where is the conflict😢

**Comment by vuslatsena** [comment]
WHERE IS THE PROBLEM ?!!!! come on netlify

**Pull request merged**